casper/src/main/scala/coop/rchain/casper/MultiParentCasperImpl.scala

```diff
@@ -65,7 +66,7 @@ class MultiParentCasperImpl[F[_]: Sync: Concurrent: Log: Time: SafetyOracle: Las
 
   //TODO: Extract hardcoded version and expirationThreshold
   private val version             = 1L
-  private val expirationThreshold = 50
+  private val expirationThreshold = Int.MaxValue
```

Comment:
@cboscolo and Rao have confirmed that we are expecting from clients to supply correct block number for `ValidAfterBlockNumber` field in a deploy.

Reply:
so are you saying that this is not necessary anymore?

Reply:
The logic should not change and threshold should stay at 50.